rubin_sim/maf/metricBundles/metricBundleGroup.py

```diff
@@ -447,7 +449,17 @@ def _runCompatible(self, compatibleList):
         else:
             cache = False
         # Run through all slicepoints and calculate metrics.
-        for slice_i in slicer:
+        if self.verbose:
+            slicer_iter = tqdm.tqdm(
+                slicer,
+                desc="Processing slices",
+                ncols=79,
+                file=sys.stdout,
+            )
+        else:
+            slicer_iter = slicer
+
+        for slice_i in slicer_iter:
```

Comment:
I can back this out if you want. It is very nice to know when the metric will finish.

Reply:
Looks like it's pretty minor addition, only gets triggered with "verbose" which shouldn't happen very often, and is a minor time increase in execution when it does. 
We can add it to the requirements.txt file and update the conda feedstock to add the package. 
(do you want to add it to the requirements.txt file here, so that the GitHub actions pass?)